src/views/collection-view/index.tsx

```diff
@@ -32,18 +34,12 @@ function CollectionView({
         numTutorials={tutorials.length}
       />
       <h2 id={layoutProps.headings[1].slug}>Tutorials</h2>
-      <ol>
```

Comment:
Just wanna say it felt really smooth switching from the placeholder component to the polished one 😍 I really appreciate the excellent stubbing work 💯